.github/workflows/pr_assign.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Auto Assign'
+on:
+    pull_request_target:
+      types: [opened, ready_for_review]
+  
+concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref }}
+    cancel-in-progress: true
+  
+jobs:
+    add-owner:
+      runs-on: ubuntu-24.04
+      if: ${{ github.actor != 'dependabot[bot]' }}
+      steps:
+        - name: run
+          uses: kentaro-m/auto-assign-action@f4648c0a9fdb753479e9e75fc251f507ce17bb7e # v2.0.0
+          with:
+            configuration-path: ".github/auto_assign.yml"
+            repo-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
this was confusing me. The spacing here is different. some indents use 2 spaces some use 4